vyked/decorators/http.py

```diff
@@ -106,6 +106,12 @@ def f(self, *args, **kwargs):
                     raise e
 
                 except Exception as e:
+                    if suppressed_errors:
+                        for _error in suppressed_errors:
+                            if isinstance(e, _error):
+                                _logger.info('Suppressed exception %s for method %s ', e.__class__.__name__,
+                                             func.__name__)
+                                return {}
```

Comment:
instead of returning {} we shoudl raise it there so that it not get logged in exceptions log